datastore/google/cloud/datastore/_onestore_v3.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
```

Comment:
To be very clear: This is not a public proto and has no guarantees about forward/backward compatibility from Cloud Datastore. Use at your own risk.